### PR TITLE
v1.2

### PR DESCRIPTION
**Pull request by lovincyrus**

### CODE REVIEW

File: components/SectionTestnets.vue [hunk 1]
```diff
@@ -20,168 +20,323 @@
...
+                  >
+                    Live
+                  </div>
+                </div>
+              </div>
+              <div class="testnets-list__item__text">
+                <div class="testnets-list__item__text__mid"></div>
+                <p
+                  class="testnets-list__item__text__mid__desc tm-rf1 tm-lh-copy"
+                  tm-measure
```
Comment (nassdonald) [inline components/SectionTestnets.vue:54]: These are outside of the class value on all of these testnet item paragraphs.

**Review by nassdonald — DISMISSED** [review]
Just a few minor fixes to do.

**Comment by nassdonald** [comment]
Oh I forgot to mention we should also remove the HackAtom V banner.

File: components/SectionTestnets.vue [hunk 2]
```diff
@@ -540,6 +690,23 @@ export default {}
   background transparent !important
   color var(--gray-600) !important
 
+.list-title
+  margin-bottom var(--spacing-5)
+
+.list-item
+  position relative
+  &::after
+    content "↗"
```
Comment (nassdonald) [inline components/SectionTestnets.vue:699]: This should be on the anchor element so it is part of the link and inherits the color etc.

**Review by nassdonald — CHANGES_REQUESTED** [review]
One last minor fix. Also, if you can remove the HackAtom V banner, that'd be great. ✨

**Comment by lovincyrus** [comment]
> One last minor fix. Also, if you can remove the HackAtom V banner, that'd be great. ✨

Remove the banner altogether or update the copy? @nassdonald

**Comment by nassdonald** [comment]
Remove it altogether, I reckon. There's not much point in promoting the hackathon now that it's pretty much over. And I don't think there's anything important enough to promote in its place at the moment.

File: components/SectionTestnets.vue [hunk 3]
```diff
@@ -782,6 +954,12 @@ export default {}
       grid-column 1 / span 12
 
 @media screen and (max-width: 576px)
+  .section-testnets .testnets-list__item__text__top__status
+    margin-left 0
+
+  .testnets-list__item__text__top__box__inactive
+    margin-top var(--spacing-4)
```
[the remark this answers is not in the document]
Comment (nassdonald) [reply components/SectionTestnets.vue]: Soooo sorry, I commented on the wrong line in a rush. This should be `--spacing-6` on `.testnets-list__item__text` wherever that's declared. Currently it's at `--spacing-4`.

**Review by nassdonald — APPROVED** [review]
🚢 🚢 🚢

**Pull request merged**